### ISSUE TITLE
UI / UX

### ISSUE THREAD

**Issue by Awzaw** [poggit/poggit#21]
Here's a few ideas, let me know your thoughts.

Guest Homepage:

The main content I think should be moved into an "About" or "FAQ" page, so that the homepage can devote as much space as possible to people getting what they want: plugins.

There should be as little dev speak ("builds", "releases"...) as possible. It should be uncluttered and simple so that even foreign kids can easily find and download a plugin. Since guests only see releases I suggest only using the term "Plugin" at least on the main release download page, and grouping them by recent, featured, recommended, category etc.

Search
An AJAX search box might be neat, again grouping results by name, author, category etc. If not, normal search is fine too of course.

Member Homepage

I need to see the timeline data before knowing how that page should be laid out, but here the main content could be focused on info about the workflow and how to use poggit, what it does, what it doesn't do etc, keeping the text short with links to information elsewhere if too long/detailed.

The other panes could display public releases and your own projects as it is now, plus the timeline underneath, for example.

Member CI page

I would change the toggle repo flow: instead of clicking, waiting for the arrow, clicking again on the arrow to display repos... one solution would be to use a jquery modal dialog overlay to prevent any user interaction until the call back populates the page, so only one click.

**Comment by archie426** [comment]
I was thinking, instead of having a plugins column on the member page, swap that for a 'recently submitted' or a 'recently released' plugins and put the actual plugins on another page

**Comment by HimbeersaftLP** [comment]
Also nice for newbies would be a "How to install plugins" section

**Comment by archie426** [comment]
@HimbeersaftLP Good idea

**Comment by SOF3** [comment]
Please remind people that there is only search for releases, no search for builds.

**Comment by SOF3** [comment]
What you are proposing about member homepage kind of duplicates what /pi does. Consider redirection or using the same module?

**Comment by Awzaw** [comment]
I'd expect the Member homepage to be a general overview with panels pretty much as you have now but different arrangement if necessary, mixing public stuff (new dev builds/releases, maybe even something social) so it looks like a  "dashboard" overview.  The member CI page could then be focused entirely on their own repo/project admin

**Comment by Magicode1** [comment]
I totally agree with this, because if you remember, some plugins on Pocketmine got hundres of thousands of downloads (https://forums.pocketmine.net/plugins/essentialspe.886/) 

Many of them weren't users and aren't very experienced. A less cluttered guest page would be the perfect solution to this.

**Comment by Magicode1** [comment]
Do we want guests to be able to download plugins from the site without logging in?

**Comment by SOF3** [comment]
@Magicode1 you mean plugin releases or CI builds?

**Comment by Magicode1** [comment]
Releases

**Comment by SOF3** [comment]
Of course. All plugins that have been approved by community or staff are available to all users.

**Comment by Magicode1** [comment]
Sweet, just wanted to make sure

**Comment by TheDiamondYT1** [comment]
I think the UI could be a *bit* nicer. Look at a plugin page on Bukkit https://dev.bukkit.org/projects/jarvis. Im not saying lets copy them, but it looks really nice. Even adding maybe a few `<hr>` tags and changing the background could make it look nicer. I would do it myself but i cant install poggit :/

**Comment by Awzaw** [comment]
There's UI improvement needed and we'll get to that in due course... but don't forget that Bukkit.org uses a CMS as it's only a plugin list, whereas Poggit was build from scratch. And page loads for plugins on Poggit are twice as fast as Bukkit ;-)

**Comment by TheDiamondYT1** [comment]
@Awzaw I know, im just giving an example of a nice looking plugin repository

**Comment by JaxkDev** [comment]
Believe this is the last item yet to be updated:
> Member CI page
> I would change the toggle repo flow: instead of clicking, waiting for the arrow, clicking again on the arrow to display repos... one solution would be to use a modal dialog overlay like you do already for the repo enable dialog, to prevent any user interaction until the call back populates the page - only one click and a nice effect.
